src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -92,27 +94,27 @@ transformed, cached, saved, or whatever), that the IDs between the two
 different data views would have any discernable relationship.
 
 Since this ID is practically often derived from the IDs of some other
-`ICounted` (for example, for a transform, the IDs of the output are usually derived
-from the IDs of the input), it is not only necessary to claim that the ID
-generated here is probabilistically unique, but also describe a procedure or
-set of guidelines implementors of this method should attempt to follow, in
-order to ensure that downstream components have a fair shake at producing
-unique IDs themselves.
+`DataViewRow`. For example, when applying `ITransformer.Transform`, the IDs of
+the output are usually derived from the IDs of the input. It is not only
+necessary to claim that the ID generated here is probabilistically unique, but
+also describe a procedure or set of guidelines implementors of this method
+should attempt to follow, in order to ensure that downstream components have a
+fair shake at producing unique IDs themselves.
 
 Duplicate IDs being improbable is practically accomplished with a
-hashing-derived mechanism. For this we have the `UInt128` methods `Fork`,
-`Next`, and `Combine`. See their documentation for specifics, but they all
-have in common that they treat the `UInt128` as some sort of intermediate hash
-state, then return a new hash state based on hashing of a block of additional
-'bits.' (Since the bits hashed may be fixed, depending on the operation, this
-can be very efficient.) The basic assumption underlying all of that collisions
-between two different hash states on the same data, or hashes on the same hash
-state on different data, are unlikely to collide. Note that this is also the
-reason why `UInt128` was introduced; collisions become likely when we have the
-number of elements on the order of the square root of the hash space. The
-square root of `UInt64.MaxValue` is only several billion, a totally reasonable
-number of instances in a dataset, whereas a collision in a 128-bit space is
-less likely.
+hashing-derived mechanism. For this we have the `DataViewRowId` methods
+`Fork`, `Next`, and `Combine`. See their documentation for specifics, but they
+all have in common that they treat the `DataViewRowId` as some sort of
+intermediate hash state, then return a new hash state based on hashing of a
+block of additional 'bits.' (Since the bits hashed may be fixed, depending on
+the operation, this can be very efficient.) The basic assumption underlying
+all of that collisions between two different hash states on the same data, or
+hashes on the same hash state on different data, are unlikely to collide. Note
```

Comment:
I don't think I understand this sentence :-).

Reply:
Yeah, I think I was trying to put it together and it didn't quite work out somehow. I'll try rephrasing -- it is kind of an annoying complex topic.